core/domain/state_domain.py

```diff
@@ -1104,6 +1104,8 @@ def traverse_by_schema_and_get(
                 not schema_utils.is_subtitled_html_schema(schema) and
                 not schema_utils.is_subtitled_unicode_schema(schema)
         ):
+            # Because a SubtitledHtml and SubtitledUnicode schema is encoded
```

Comment:
This comment still doesn't explain why we need to exclude them. What's wrong with including them?

Reply:
The problem is that while they are encoded as dicts, they are actually SubtitledHtml/SubtitledUnicode domain objects, and traversing on them would result in a AttributeError in this branch, since they don't have __getitem__ (they aren't actually dicts). I've updated the docstr here.

Reply:
Changed back to custom schema - Done.